### PR TITLE
[SDK-CORE] Added FlowOperator Query and its Entity

### PR DESCRIPTION
**Pull request by msoni89**
Added, account access settings and query required for Permission and allowance feature (Dashboard).

### CODE REVIEW

**Comment by hellwolf** [comment]
mark it as draft instead of "DO NOT MERGE"

File: packages/sdk-core/src/subgraph/entities/accountAccessSetting/accountAccessSetting.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+export interface AccountAccessSetting {
+    id: string;
+    createdAtBlockNumber: BlockNumber;
+    createdAtTimestamp: Timestamp;
+    updatedAtBlockNumber: BlockNumber;
+    updatedAtTimestamp: Timestamp;
+
+    flowOperator: string;
+    sender: {
```
Comment (kasparkallas) [inline packages/sdk-core/src/subgraph/entities/accountAccessSetting/accountAccessSetting.ts]: `sender: Address`

File: packages/sdk-core/src/subgraph/entities/accountAccessSetting/accountAccessSetting.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        id: Address;
+    };
+    token: {
```
Comment (kasparkallas) [inline packages/sdk-core/src/subgraph/entities/accountAccessSetting/accountAccessSetting.ts]: token: `Address`

File: packages/sdk-core/src/subgraph/entities/accountAccessSetting/accountAccessSetting.ts [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    AccountAccessSetting,
+    AccountAccessSettingListQuery,
+    AccountAccessSettingsQuery,
+    AccountAccessSettingsQueryVariables
+> {
+    getAddressFieldKeysFromFilter = (): {
+        accountKeys: (keyof FlowOperator_Filter)[];
+        tokenKeys: (keyof FlowOperator_Filter)[];
+    } => ({
+        accountKeys: ["sender"],
```
Comment (kasparkallas) [inline packages/sdk-core/src/subgraph/entities/accountAccessSetting/accountAccessSetting.ts]: `flowOperator` too

**Review by kasparkallas — APPROVED** [review]

**Pull request merged**